server.js

```diff
@@ -148,6 +146,23 @@ app.get('/init', async (request, response) => {
   }
 });
 
+app.get('/customers/:sessionId', async (request, response) => {
+  const sessionId = request.params.sessionId;
+
+  try {
+    const customers = await Promise.all(
+      storeCustomers.map(async (customer) => {
```

Comment:
no await, no need for async

Reply:
I'm not sure why - but if I delete this, then everything stops (I'm waiting forever for customers). There is not any error, but I'm still waiting.